modules/steam.nix

```diff
@@ -340,6 +340,9 @@ in
         # Disable automatic audio device switching in steam, now handled by wireplumber
         STEAM_DISABLE_AUDIO_DEVICE_SWITCHING = "1";
 
+        # Let steam know it can unmount drives without superuser privileges
+        STEAM_ALLOW_DRIVE_UNMOUNT = "1";
+
```

Comment:
I've not seen any different behaviour from Steam with this on. Vendor added this previously.

I suppose Steam has to handle the mounting to get this going? Or maybe it needs to be mounted using udisks? My SD card is auto-mounted by systemd on access [assuming the right one is inserted].